src/vulkan/device.cc

```diff
@@ -354,20 +355,22 @@ Device::Device(VkInstance instance,
 
 Device::~Device() = default;
 
-Result Device::LoadVulkanPointers(
-    PFN_vkGetInstanceProcAddr getInstanceProcAddr) {
+Result Device::LoadVulkanPointers(PFN_vkGetInstanceProcAddr getInstanceProcAddr,
+                                  Delegate* delegate) {
+  (void)delegate;  // work around -Werror=unused-parameter
```

Comment:
What about if we stick right here something like:
```
if (delegate && delegate->LogGraphicsCalls())
  delegate->Log("Loading Vulkan Pointers");
```
Then the delegate is used, and we get the context of these calls as well?